include/wabt/binary-reader.h

```diff
@@ -59,6 +59,13 @@ struct TypeMut {
 };
 using TypeMutVector = std::vector<TypeMut>;
 
+struct RawCatch {
+  CatchKind kind;
+  Index tag;
+  Index depth;
+};
+using RawCatchVector = std::vector<RawCatch>;
```

Comment:
Why is "Raw" in the name here?

Reply:
they match what's in the binary, vs the representation used in the wabt IR.

Reply:
But we do we the term "Raw" to mean "in the binary" elsewhere in this file?   i.e. what do we do for other types that have binary and wabt IR representations?

Reply:
there are... none. the only other type that's slightly out of place is TypeMut. we'll see if we can come up with a better name...